### PR TITLE
Qh 18 replace cookie identifier

### PR DESCRIPTION
**Pull request by atalive**
Contains the following tickets 
[QH-11 - End-to-end test setup](https://liveintent.atlassian.net/secure/RapidBoard.jspa?rapidView=445&projectKey=QH&modal=detail&selectedIssue=QH-11)
[QH-12 - Introduce Data Provider](https://liveintent.atlassian.net/secure/RapidBoard.jspa?rapidView=445&projectKey=QH&modal=detail&selectedIssue=QH-12)
[QH-18 - Replace Cookie identifier](https://liveintent.atlassian.net/secure/RapidBoard.jspa?rapidView=445&projectKey=QH&modal=detail&selectedIssue=QH-18)

### CODE REVIEW

File: e2e/converter.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+ *
+ * @param {Object} pixelUrl
+ * @param {String} pixelPath
+ * @return {*[]}
+ */
+function getValueByKey(pixelUrl, pixelPath) {
+    let value
+    let current = _.get(pixelUrl, pixelPath)
+
+    if (pixelPath === 'params.ext_s_vi') {
```
Comment (gipeshka) [inline e2e/converter.js:40]: what is this logic please? if `ext_s_vi` is present, then read value from it and from `muuid `? What if `muuid` is there, but `ext_s_vi` isn't?

File: live-pixel/converter.js [hunk 2]
```diff
@@ -44,6 +57,58 @@ function urlToPixelEvent(url) {
...
+    return formatPixelEvent(pixelEvent, domainUserIdIsTheRightFormat)
+}
+
+/**
+ *
+ * @param {Object} url
+ * @param {Boolean} condition
+ * @return {Object}
+ */
+function getValueByCondition(url, condition) {
```
Comment (gipeshka) [inline live-pixel/converter.js:74]: should the name reflect the nature of the `Value`?

File: test/end-to-end.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        let identifier = lpUrl.params.ext_s_vi
+
+        const pixelEvent = pixelStreamClient.getData(pixelRecordPredicate(identifier))
+        const midasEvent = midasStreamClient.getData(midasRecordPredicate(identifier))
+        const makeRequest = lpClient.makeRequest(lpUrl)
+
+        let valuesArray = await Promise.all([makeRequest, pixelEvent, midasEvent])
+        let midasRow = valuesArray.pop()
+
+
```
Comment (gipeshka) [inline test/end-to-end.js:43]:
```suggestion
```

File: live-pixel/converter.js (hunk 2 above)
Comment (atalive) [reply live-pixel/converter.js:74]: yes but not sure what it could be.

**Pull request closed without merging**